src/main/java/talkabot/exceptions/InvalidEditException.java

```diff
@@ -0,0 +1,14 @@
+package talkabot.exceptions;
+
+/**
+ * InvalidEditException class to signal wrong task input given by the user to edit.
+ */
+public class InvalidEditException extends TalkaBotException {
+
+    /**
+     * Constructor for InvalidEditException class.

```

Comment:
Should this be "Constructs a InvalidEditException class..." instead? 
```suggestion
     * Constructs a InvalidEditException class...

```